MAINTAINERS.md

```diff
@@ -0,0 +1,199 @@
+
+- **Identifies Relevant Code:** The relevant code and tests to be changed should be linked in the issue.
+
+- **Ready to Test:** There should be existing tests that can be modified, or existing test cases fit to be copied. If the area of code doesn't have tests, before labeling the issue, add a test fixture. This prep often makes a great help wanted task!
+
+Maintainers are encouraged to invest the extra effort to write and shepherd along these issues and PRs, since onboarding reliable, long-term contributors is an excellent foundation for a sustainable, long-running project that doesn’t burn out the participants.
+
+**Measures can include:**
+- Help with the contributor’s dev setup, and if you realise that the onboarding documentation for developers is lacking or out of date, assign their improvement to the new contributor! They probably took notes on the process anyway.
+- Proactively watch open first timer-PRs as they change and get in touch if you can save people some time or pain.
+- Invite them to meetings and chats
+- Rope in other maintainers for a second LGTM at the end
+- If the contributor agrees, publicly thank them on whichever channels you broadcast on, as well as in the release notes
+- Suggest related `Help wanted` issues
+
+Unless the contributors are insecure about public communication, you’re encouraged to _not_ use private messages as much as possible. Keeping communication public ensures that other people can find and benefit from your discussions in the future.
+
+#### Other Issue Types
+
+There are a variety of additonal labels that are generally self-explanatory, for example:
+
+- `Feature`
+- `Documentation`
+- `Performance`
+- `Testing`
+- `Cleanup`
+- `Security`
+- `Compatibility`
+
+A full list can always be found on the project’s [labels page](https://github.com/openpgpjs/openpgpjs/labels).
+
+### Step 2: Prioritize the Issue
+
+Aside from a type label and the housekeeping labels, issues that are or should become actionable also benefit from having a priority label.
+
+#### `priority/critical-urgent`
+
+Something has gone 💥. Stuff is on fire. Drop what you’re doing and work on this next. Must be in the next planned release, or requires its own release.
+
+#### `priority/important-soon`
+
+Must be staffed and worked on either currently or very soon—ideally in time for the next release. Important, but wouldn't block a release.
+
+#### `priority/important-longterm`
+
+Important over the long term, but may not be currently staffed and/or may require multiple releases to complete. Wouldn't block a release.
+
+#### `priority/backlog`
+
+Would be nice, but no one has the resources to work on it. Good candidates for the `Help wanted` label.
+
+### Step 3: Assign the Work
+
+It should be clear who is responsible for working on an issue once it has been triaged and deemed actionable. This also includes eventual reviews. Ideally don’t assign things to people who aren’t expecting it.
```

Comment:
The above also means that this isn't always possible. If one of the maintainers will do it, then yes, but otherwise it's up to the applications and contributors to pick up the work, not for us to assign it.